### PR TITLE
add WS2022 LTSC, rm WS2004 sac, bump versions

### PR DESCRIPTION
**Pull request by rosskirkpat**
- bump to dapper 0.5.8
- remove windows server 2004 SAC from drone build steps and agent manifest templates
- bump windows kubectl to 1.21.3 in package/windows/Dockerfile.agent
- bump to go1.16.9 in Dockerfile.dapper
- Add Windows Server 2022 LTSC to drone build steps and agent manifest templates
- bump drone-docker-image-digests to 0.0.12 in preparation for 2022 being added and a new release being cut
- go mod tidy

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -2,4 +2,5 @@
 # line endings. Windows users contributing to Go will need to use a
 # modern version of git and editors capable of LF line endings.
 
-*.go -text diff=golang
\ No newline at end of file
+*.go -text diff=golang
+*.go text eol=lf
```
Comment (luthermonson) [inline .gitattributes:6]: you mean to check this in?

File: .drone.yml [hunk 2]
```diff
@@ -1205,58 +1086,6 @@ depends_on:
...
-  arch: amd64
-  version: 2004
-
-# remove this and use upstream images when https://github.com/drone/drone-git/pull/25 is merged
-clone:
-  disable: true
-
-steps:
-- name: clone
-  image: luthermonson/drone-git:windows-2004-amd64
```
Comment (luthermonson) [inline .drone.yml:1223]: the faster we get this guy out of the code base the better.

File: .gitattributes (hunk 1 above)
Comment (rosskirkpat) [reply .gitattributes:6]: Yes, as we have developers using Windows and this ensures that go files won't have CRLF line endings

File: .drone.yml (hunk 2 above)
Comment (rosskirkpat) [reply .drone.yml:1223]: 20h2 removal will be coming soon enough

**Comment by luthermonson** [comment]
bump to using v0.5.8 dapper and add the 2022 pipeline and we'll be complete on rancher/rancher

**Review by phillipsj — DISMISSED** [review]
LGTM

**Review by luthermonson — APPROVED** [review]

**Review by phillipsj — APPROVED** [review]
LGTM

**Review by luthermonson — APPROVED** [review]

**Pull request merged**